RecoLocalTracker/SiPixelRecHits/interface/VVIObjF.h

```diff
@@ -32,7 +34,7 @@ class VVIObjF {
 private:
   // Vavilov distribution parameters (inputs and common block /G116C1/)
 
-  const int mode_;  //!< set to 0 to calculate the density function and to 1 to calculate the distribution function
+  const int mode_ = 1;  //!< set to 0 to calculate the density function and to 1 to calculate the distribution function
```

Comment:
why does it have to be set here?

Reply:
It can be initialized in the initializer list of the new constructor. The old one already initializes it taking one of its arguments. I don't see duplication.

Reply:
I'll fix this after the current tests have finished